experimental/segmenter/src/line.rs

```diff
@@ -171,17 +174,42 @@ pub type LineBreakIteratorUtf16<'l, 's> = LineBreakIterator<'l, 's, LineBreakTyp
 ///
 /// [LD3]: https://www.unicode.org/reports/tr14/#LD3
 /// [LD7]: https://www.unicode.org/reports/tr14/#LD7
+/// [LB3]: https://www.unicode.org/reports/tr14/#LB3
+/// [LB4]: https://www.unicode.org/reports/tr14/#LB4
+/// [LB5]: https://www.unicode.org/reports/tr14/#LB5
 ///
 /// ```rust
 /// use icu_segmenter::LineSegmenter;
 ///
 /// let segmenter = LineSegmenter::try_new_auto_unstable(&icu_testdata::unstable())
 ///     .expect("Data exists");
 ///
-/// let breakpoints: Vec<usize> =
-///     segmenter.segment_str("Summary\r\nThis annex…").collect();
+/// let text = "Summary\r\nThis annex…";
+///
+/// let breakpoints: Vec<usize> = segmenter.segment_str(text).collect();
 /// // 9 and 22 are mandatory breaks, 14 is a line break opportunity.
 /// assert_eq!(&breakpoints, &[9, 14, 22]);
+///
+/// use icu::properties::{maps, LineBreak};
```

Comment:
nit: To avoid people copying this entire code I would likely make this a separate example, contextualized with "if you need to know the mandatory breaks here's what you can do".

Note that in Rust doctests you can hide setup lines by using `#`